modules/machine-lifecycle-hook-deletion-format.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * machine_management/deleting-machine.adoc
+
+:_content-type: REFERENCE
+[id="machine-lifecycle-hook-deletion-format_{context}"]
+= Deletion lifecycle hook configuration
+
+The following YAML snippets demonstrate the format and placement of deletion lifecycle hook configurations within a machine set:
+
+.YAML snippet demonstrating a `preDrain` lifecycle hook
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
```

Comment:
Probably better to be on the Machine rather than MachineSet for all the examples